src/test/resources/features/P2SubAgentUserJourney.feature

```diff
@@ -0,0 +1,87 @@
+@p2tests
+Feature: Pillar2 submission Agent User Journey
+  As an Agent authenticated user
+  I should be able to access Agent features
+
+  @p2zap_p2accessibility
+  Scenario: 1 - Agent user capturing Pillar2 ID
+    Given Agent User logs in with existing entity group HMRC-AS-AGENT, AgentReference and 1234 for Pillar2 submission service
+    And I add delegated enrolment with HMRC-PILLAR2-ORG, PLRID, XMPLR0012345674 and pillar2-auth for Pillar2 submission service
+    Then I should be navigated to ASA Pillar2 Input Page of Pillar2 Submission
+    And I click on Continue button
+    Then I should see an error message Enter your client’s Pillar 2 top-up taxes ID on the Agent enter pillar2 id Page
```

Comment:
content should be covered as part of unit tests

Reply:
Removed Content Validation.